tests/cypress/support/commands.js

```diff
@@ -530,6 +529,28 @@ Cypress.Commands.add('writeFilterValue', (clear, filterValue) => {
         });
 });
 
+Cypress.Commands.add('selectFilterValue', (clear, filterValue) => {
+    let filterValues = [];
+    if (clear) {
+        cy.get('.cvat-annotations-filters-input').within(() => {
+            cy.get('.ant-select-selection-item-remove').click();
+        });
+    }
+    cy.get('body').click();
+    cy.get('.cvat-annotations-filters-input').click();
+    cy.contains('.cvat-annotations-filters-input-history-element', filterValue).scrollIntoView().click();
+    cy.get('body').click();
+    cy.get('.cvat-annotations-filters-input').within(() => {
+        cy.get('.ant-select-selection-item-content')
```

Comment:
Cannot we just check if ``.cvat-annotations-filters-input`` contains ``filterValue``? I am not sure why we need this additional array ``filterValues``.

Reply:
Yes, we can. Unnecessary actions removed.